#request
string str1
---
#response
string str2